chapters/2-document-creation-information.md

```diff
@@ -315,7 +315,7 @@ Example:
 
 **2.10.4** Data Format: Free form text that can span multiple lines.
 
-In `tag:value` format this is delimited by `<text> .. </text>`, in RDF, it is delimited by `<rdfs:comment>`.
```

Comment:
Same comment as above - I won't comment on all of the changes, just revert the changes for ```rdfs:comment```